src/@types/rlp-encoding.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'rlp-encoding' {
```

Comment:
If someone could enlighten me why we are using `rlp-encoding` lib in this repository instead of `rlp`, the latter has types now, but when I switched I was getting some decoding issues for "invalid remainder" - not sure if it's an overly strict check or what.

Reply:
See #49